src/components/LandingPage/LandingPage.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import {shallow, ShallowWrapper} from 'enzyme';
+import * as React from 'react';
+import LandingPage from "./LandingPage";
+import CreateGameService from './CreateGameService';
+import StarishWarsState from '../StarishWarsState';
+import ILandingPage from './LandingPageInterface';
+
+
+describe('LandingPage', () => {
+    //let component: import("../../../node_modules/@types/enzyme/index").ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;
+    let component: ShallowWrapper<ILandingPage, StarishWarsState>;
+    let fakeCreateGameService: CreateGameService
+    let myState : StarishWarsState
+
+    beforeEach(() => {
+        fakeCreateGameService = new CreateGameService(); 
+        fakeCreateGameService.createGame = jest.fn().mockReturnValue(1);
+        myState = {gameId:1}
+        component = shallow(<LandingPage createGameService={fakeCreateGameService} mystate={myState} />);
+    });
+
+    it('renders a create game button', () => {
+        expect(component.find('.CreateGameButton')).toHaveLength(1);
+    });
+    it('renders a button with the text Start Game on it', () => {
+        expect(component.find('.CreateGameButton').first().text()).toBe("Start Game")
+    });
+    it("clicking button calls backend to create game", () => {
+        component.find('button').simulate('click');
+        expect(fakeCreateGameService.createGame).toHaveBeenCalled()
+    });
+    it("has a game id in state store after button is clicked", () => {
+        component.find('button').simulate('click');
+        expect(component.state().gameId).toBeGreaterThan(0);
+    });
```

Comment:
See previous comments about gameId. It'll be a string, not a number, so need to check it's not empty or something similar.